### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by orome** [dotintent/react-native-ble-plx#968]
Can I run this in Expo if I am using an emulator (e.g., [react-native-blemulator](https://github.com/Polidea/react-native-blemulator))

**Comment by savety6** [comment]
Yes, you can run it by following this guide [,](https://expo.canny.io/feature-requests/p/bluetooth-1), but you can't use it with an emulator or the "Expo Go" app! You have to build your test build.
